torchrl/collectors/collectors.py

```diff
@@ -809,7 +812,7 @@ def _step_and_maybe_reset(self) -> None:
             traj_ids[traj_done_or_terminated] = traj_ids.max() + torch.arange(
                 1, traj_done_or_terminated.sum() + 1, device=traj_ids.device
             )
-            self._tensordict.set_(
+            self._tensordict.set(
                 ("collector", "traj_ids"), traj_ids
             )  # no ops if they already match
```

Comment:
Will there still be a case where this is np ops?

Reply:
no we can remove that comment